app/src/main/res/values/strings.xml

```diff
@@ -34,7 +34,7 @@
   <string name="prayed_for_text">You have prayed for</string>
   <string name="prayer_reminders">Reminder:</string>
   <string name="number_of_people_each_day">Number of people to pray for daily:</string>
-  <string name="reminder_time_picker_title">Prayer reminder time:</string>
+  <string name="reminder_time_picker_title">Prayer reminder time: </string>
```

Comment:
Why was the space added?
It might not map right for other translations.

Reply:
After I combined the notification time to display on the label (instead of a separate widget), the visual alignment was affected. I will see if I can fix that. Either way, I will remove the space.